From 2b825a1748b2ecd624a38976d1bd6601bdb198c9 Mon Sep 17 00:00:00 2001
From: Ronny Kunze <Shinji.Ikari_de@gmx.de>
Date: Sat, 4 Aug 2018 16:53:06 +0200
Subject: [PATCH] rearange CMake find_package OpenSSL

the command seems to be malformed . The keyword VERSION is not valid .
And the version should follow direct after the packagename.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f8b8330f2..f1d07eea2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -183,7 +183,7 @@ if(BUILD_CLIENT)
     endif()
     find_package(Sphinx)
     find_package(PdfLatex)
-    find_package(OpenSSL REQUIRED VERSION 1.1)
+    find_package(OpenSSL 1.1 REQUIRED )
 
    find_package(ZLIB REQUIRED)
    find_package(GLib2)
